docs/hardware/devices/techo/buttons.mdx

```diff
@@ -7,14 +7,22 @@ sidebar_position: 1
 
 ## Functionality
 
-- **Capacitive Touch Button (Top):**
-  - **Short press:** Updates the e-ink display.
+- **Capacitive Touch Button (Top):**<br />
+  *If "Wake on Tap or Motion" setting enabled:*
+  - **Touch:** 
+    - Update the display
+    - *(at screensaver)* Wake the display.
 - **Reset Button (Button 1):**
-  - **Single press:** Resets the device.
-  - **Double press:** Puts the device into bootloader mode which allows you to update the firmware.
-- **Program/Power Button (Button 2):**
-  - **Single press:** Changes the information page displayed on the device's screen.
-  - **Double press:** Turns the screen backlight on/off and sends an adhoc ping of the device's position to the network.
-  - **Long press:** Signals the device to shutdown after 5 seconds.
+  - **Single press:** Power-on / reboot.
+  - **Double press:** Enter bootloader mode, for firmware update.
+- **Program Button (Button 2):**
+  - **Single press:** 
+    - Display next page of information.
+    - *(at screensaver)* Wake the display.
+    - *(when off)* Enter bootloader mode, for firmware update.
```

Comment:
Is this mode any different to a double-press of the top button?

Reply:
Exactly the same mode, just reached by a keypress. More of a nuisance than a feature. I'm not sure at what point this behavior changed, but my T-Echo has always done this. I confirmed the behavior yesterday with another user.

Is there a different way you would suggest structuring the info?

Reply:
Just curious, I haven't dusted off the ol' T-echo in a while.